src/components/crewvariants.tsx

```diff
@@ -17,33 +17,68 @@ class CrewVariants extends Component<CrewVariantsProps, CrewVariantsState> {
 	};
 
 	componentDidMount() {
+		// Get variant names from traits_hidden
+		let ignore = [
```

Comment:
Do we have some mechanism for identifying new traits when they first appear so we can determine if they need to be added to this list?

Something to build into the "misc scripts" that handle crew updates, perhaps?

Reply:
@ineffyble & @ussjohnjay I've invited you both to the stt-datacore organisation so you should then be able to see the scripts repository.  I've had a little delve into it, and have no idea what's going on, or how it works.  If you can't work it out, you'll need to ask @TemporalAgent7 for help.